scripts/common.js

```diff
@@ -0,0 +1,61 @@
+const fs = require("fs");
```

Comment:
I take it these new scripts are factored out of `index.js`? Were there any changes or can I just trust that the logic is the same?

Reply:
Changes are just that instead of execing npm we use wireit for task scheduling. So index.js previously ran the build command in each library, then verified the result files against goldens and copied them into the docs dir, then after doing that for all of the libraries, it ran `npm run build` in the docs directory.

Now this is just a copy-and-verify-results script which depends on all of the libraries' builds, and ./docs:build depends on copy-and-verify-results

Also, broke the set of libraries out into common.js so that we could depend on that from the install script